generate.py

```diff
@@ -179,8 +192,20 @@ def main():
 
         # Run the WaveNet to predict the next sample.
         prediction = sess.run(outputs, feed_dict={samples: window})[0]
+
+        # Scale prediction distribution using temperature.
+        np.seterr(divide='ignore')
+        scaled_prediction = np.log(prediction) / args.temperature
+        scaled_prediction = scaled_prediction - np.logaddexp.reduce(scaled_prediction)
+        scaled_prediction = np.exp(scaled_prediction)
+        np.seterr(divide='warn')
+
+        # Prediction distribution at temperature=1.0 should be unchanged after scaling.
+        if args.temperature == 1.0:
```

Comment:
You didn't find it because it doesn't exist yet -- there is a PR that provides tests for this file. Thanks for specifying how to test it, though! We'll move it later, no problem.